docs/dev.adoc

```diff
@@ -112,29 +112,75 @@ directory in dataplane-operator.
 
 kuttl tests are under the
 https://github.com/openstack-k8s-operators/dataplane-operator/tree/main/tests/kuttl/tests['tests/kuttl/tests']
-in dataplane-operator.
+in dataplane-operator and can be executed either directly from dataplane operator repository, or from https://github.com/openstack-k8s-operators/install_yamls[install_yamls]
 
 ==== Requirements
 
 The kuttl tests require a running cluster with
-https://github.com/openstack-k8s-operators/openstack-ansibleee-operator[openstack-ansibleee-operator], https://github.com/openstack-k8s-operators/openstack-baremetal-operator[openstack-baremetal-operator] and https://github.com/openstack-k8s-operators/infra-operator[infra-operator]
-running in the cluster.
+https://github.com/openstack-k8s-operators/openstack-ansibleee-operator[openstack-ansibleee-operator], https://github.com/openstack-k8s-operators/openstack-baremetal-operator[openstack-baremetal-operator], https://github.com/openstack-k8s-operators/infra-operator[infra-operator]
+and https://github.com/openstack-k8s-operators/rabbitmq-cluster-operator[rabbitmq-cluster-operator] running in the cluster.
+The easiest way to ensure presence of these operators is to use install_yamls. However, it is possible to deploy them manually as well.
 
 ==== From install_yamls
 
-The kuttl tests are run from the
+First clone both the dataplane-operator and the install-yamls repos and make sure that they are in a state (commit) you want to test.
+For the purposes of this guide, we will assume following directory layout:
+
+....
+~
+├── dataplane-operator
+└── install_yamls
+....
+
+[NOTE]
+====
+The kuttl tests are also run from the
 https://github.com/openstack-k8s-operators/install_yamls[install_yamls]
 repository by the CI jobs.
+====
+
+We will also assume that there is no cluster already running. If you already have a cluster, skip the following step.
+
+Initialize your CRC with https://github.com/openstack-k8s-operators/install_yamls?tab=readme-ov-file#deploy-dev-env-using-crc-edpm-nodes-with-isolated-networks[install_yamls] utilities.
+Make sure to have your pull secret ready and to fulfill all HW requirements. 
+CRC can be initialized without install_yamls utilities. However, using existing scripts will simplify debugging and issue reporting.
 
-Running from `install_yamls`:
+Ensure presence of dataplane operator requirements and cleanup all remains of possible previous kuttl test run.
 
 [,sh]
 ----
-cd install_yamls
-# Set environment variables if needed to use a specific repo and branch of dataplane-operator
-export DATAPLANE_REPO=https://github.com/openstack-k8s-operators/dataplane-operator.git
-export DATAPLANE_BRANCH=main
-make dataplane_kuttl
+cd ..
+make dataplane_kuttl_prep
+make dataplane_cleanup
+----
+
+Moving to the dataplane-operator repo, we can start the dataplane operator itself. 
+
+NOTE: Ports 8080 and 8081 are used for collection of metrics and health information. Make sure  that they are not in use.
```

Comment:
you can override thtat via env vars, https://github.com/openstack-k8s-operators/dataplane-operator/blob/b32e1575dd484f5681833954a55d971038a7f0ea/Makefile#L165-L166

Reply:
Good point.

Reply:
Note expanded.